src/components/App.vue

```diff
@@ -3,5 +3,18 @@
     <main>
       <router-view />
     </main>
+    <footer>
+      <site-footer />
```

Comment:
I am curious why the `<footer>` tag wraps this component instead of just being included in it.